### PR TITLE
Support GraphQL files

### PR DESCRIPTION
**Pull request by vlad-zhukov**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -86,24 +86,30 @@ function format(options) {
   )
 
   const isCss = /\.(css|less|scss)$/.test(filePath)
+  const isHtml = /\.html$/.test(filePath)
+  const isTypeScript = /\.(ts|tsx)$/.test(filePath)
```
Comment (zimme) [inline src/index.js]: `const isTypeScript = /\.tsx?$/.test(filePath)` what was wrong with this?

File: src/index.js [hunk 2]
```diff
@@ -86,24 +86,30 @@ function format(options) {
...
+    formattingOptions.prettier.parser = 'graphql'
+  } else if (isJson) {
+    formattingOptions.prettier.parser = 'json'
+    formattingOptions.prettier.trailingComma = 'none'
   }
 
   const prettify = createPrettify(formattingOptions.prettier, prettierPath)
 
-  if (isCss || isJson) {
+  if (isCss || isHtml || isTypeScript || isGraphQL || isJson) {
```
Comment (zimme) [inline src/index.js]: Typescript files shouldn't be picked up here, they should continue on to eslint.

File: src/index.js (hunk 1 above)
Comment (vlad-zhukov) [reply src/index.js]: Nothing, just copied over from [Prettier](https://github.com/prettier/prettier/blob/655fef1bf916361669aa926eb4ead875d6a2c993/src/options.js#L33-L43). Can change back if you want.

**Comment by zimme** [comment]
I know GraphQL is officially supported by prettier. I don't however see any indication that HTML is actually "released". I see that there is a parser `parse5` which parses HTML but I don't think we should add support for HTML until it's "official".

**Comment by vlad-zhukov** [comment]
I see. Personally I don't need it and I am fine with focusing only on GraphQL in this PR. If someone needs an HTML support they can open another PR.

**Comment by zimme** [comment]
Let's limit this PR to GraphQL and do a separate PR for HTML once we can confirm it's supporter officially in prettier.

Let's remove HTML related stuff from the PR and remove `isTypescript` from the check I mentioned earlier so that typescript files continues on to eslint.

**Comment by vlad-zhukov** [comment]
Updated

**Comment by azz** [comment]
Yep html is still in development :)

**Pull request merged**